TF_Service_and_Tooling_Unity/Assets/TouchFree/ServiceShared/Scripts/Configuration/Managers/TFAppConfig.cs

```diff
@@ -52,7 +52,7 @@ public class ConfigurationState
 
         // CTI Settings
         public bool ctiEnabled = false;
-        public string ctiFilePath = Path.Combine(Application.streamingAssetsPath, "CallToInteract", "AirPush_Portrait.mp4");
+        public string ctiFilePath = Path.Combine(Application.streamingAssetsPath, "CallToInteract/AirPush_Portrait.mp4");
```

Comment:
Why is this change necessary? Isn't the result the same either way?

Reply:
Well, both work as paths but it removes one of the double slashes we get in the config file, for some reason Path.Combine likes to add extra slashes